### PR TITLE
modularity on vuex and querys

### PR DESCRIPTION
**Pull request by edgarcheverier**

### CODE REVIEW

File: src/containers/Login/Login.vue [hunk 1]
```diff
@@ -63,7 +64,6 @@ export default {
         this.$store.commit('customerInformation', dude);
```
Comment (marlon-becker) [inline src/containers/Login/Login.vue]: do you think dude is a serious variable name?

File: src/store/actions.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import gql from 'graphql-tag';
+
+import graphqlClient from '../graphql';
+import { GET_RESTAURANT } from '../querys/querys';
+
+export const actions = {
+  async getDataRest({ commit }, id) {
+    let response = await graphqlClient.query({
+      query: gql`
+        ${GET_RESTAURANT}
```
Comment (marlon-becker) [inline src/store/actions.js:10]: nice :)

**Review by marlon-becker — CHANGES_REQUESTED** [review]

File: src/containers/Login/Login.vue (hunk 1 above)
Comment (edgarcheverier) [reply src/containers/Login/Login.vue]: why not ?

File: src/store/actions.js (hunk 2 above)
Comment (edgarcheverier) [reply src/store/actions.js:10]: thank you :)

**Review by marlon-becker — APPROVED** [review]

**Pull request merged**